query/src/main/scala/filodb/query/PromCirceSupport.scala

```diff
@@ -13,6 +13,10 @@ object PromCirceSupport {
     case h @ HistSampl(t, b) => Json.fromValues(Seq(t.asJson, b.asJson))
     case m @ MetadataSampl(v) => Json.fromValues(Seq(v.asJson))
     case l @ LabelSampl(v) => Json.fromValues(Seq(v.asJson))
+    // Where are these used? added to make compiler happy
```

Comment:
At least in the `src` directories of both repos, `grep` comes up empty.

Reply:
Its implicit so all runtime magic for ``io.circe``  code, I don't know what's a good way to test this